tf2onnx/tfonnx.py

```diff
@@ -690,6 +690,18 @@ def sign_op(ctx, node, name, args):
     return nodes
 
 
+def sign_op9(ctx, node, name, args):
+    # Currently supported: `float32`
+    # Ignored: `bfloat16`
+    # TODO: add support for `int32`, `int64`
+    node_dtype = ctx.get_dtype(node.output[0])
+    utils.make_sure(node_dtype, "Dtype of {} is None".format(node.name))
+    if node_dtype in [onnx_pb.TensorProto.BOOL, onnx_pb.TensorProto.FLOAT16,
+                      onnx_pb.TensorProto.COMPLEX64, onnx_pb.TensorProto.COMPLEX128]:
```

Comment:
float16 is supported according to onnx spec, remove it from here plz.
i mean we need to implement conversion logic based on onnx spec, but we may not need to test float16 for now,